integration-tests/bot/tests/test_watch_only_wallet.py

```diff
@@ -0,0 +1,29 @@
+import time
+from chainrpc import RPC
+from uuid import uuid1
+
+rpc = RPC()
+
+
+def test_watch_only_wallet(addresses):
```

Comment:
could this test be easily added to one of the integration test pipelines (either or Travis or Drone)? If not, we can do it in another PR

Reply:
> could this test be easily added to one of the integration test pipelines (either or Travis or Drone)? If not, we can do it in another PR

I'll consult @leejw51

Reply:
I've added it to the ci script, but can't test locally because of the docker image building failure, maybe try my luck on the CI server.

Reply:
It seems no luck with this, I'll revert this now, maybe try again another time.